notebooks/107-speech-recognition-quantize-wav2vec2/107-speech-recognition-quantize-wav2vec2.ipynb

```diff
@@ -0,0 +1,703 @@
+    "if fp_results:\n",
+    "    print(\"FP16 model results:\")\n",
+    "    for name, value in fp_results.items():\n",
+    "        print(f\"{name}: {value:.5f}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a05b3999",
+   "metadata": {
+    "id": "hPj_fcDAG8xG"
+   },
+   "outputs": [],
+   "source": [
+    "# Step 7: Execute the pipeline\n",
+    "print(f\"Quantizing model with {algorithms[0]['params']['preset']} preset and {algorithms[0]['name']}\")\n",
+    "start_time = time.perf_counter()\n",
+    "compressed_model = pipeline.run(model=model)\n",
+    "end_time = time.perf_counter()\n",
+    "print(f\"Quantization finished in {end_time - start_time:.2f} seconds\")\n",
+    "\n",
+    "# Step 8 (Optional): Compress model weights to quantized precision\n",
+    "#                    in order to reduce the size of final .bin file\n",
+    "compress_model_weights(model=compressed_model)\n",
+    "\n",
+    "# Step 9: Save the compressed model to the desired path\n",
+    "compressed_model_paths = save_model(model=compressed_model, save_path=MODEL_DIR, model_name=\"quantized_wav2vec2_base\")\n",
+    "compressed_model_xml = compressed_model_paths[0][\"model\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b05dc0de",
+   "metadata": {
+    "id": "hPj_fcDAG8xG"
+   },
+   "outputs": [],
+   "source": [
+    "# Step 10 (Optional): Evaluate the compressed model and print the results\n",
+    "int_results = pipeline.evaluate(model=compressed_model)\n",
+    "\n",
+    "if int_results:\n",
+    "    print(\"INT8 model results:\")\n",
+    "    for name, value in int_results.items():\n",
+    "        print(f\"{name}: {value:.5f}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "754b4d84",
+   "metadata": {},
+   "source": [
+    "## Model Usage Example with Inference Pipeline\n",
+    "Both initial (FP16) and quntized (INT8) models are exactly the same in the use\n",
+    "\n",
+    "We start with taking one example from the dataset to show inference steps for it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17bd5485",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "audio = LibriSpeechDataLoader.read_flac(f'{DATA_DIR}/LibriSpeech/test-clean/121/127105/121-127105-0017.flac')\n",
+    "\n",
+    "ipd.Audio(audio, rate=16000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7538271f",
+   "metadata": {},
+   "source": [
+    "Next, we load quantized model to the inference pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0431ac4f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "core = Core()\n",
+    "\n",
+    "model = core.read_model(compressed_model_xml)\n",
+    "compiled_model = core.compile_model(model, 'CPU')\n",
+    "infer_request = compiled_model.create_infer_request()"
```

Comment:
Please use single line inference
`compiled_model(feed_dict)[output_layer]`

Reply:
@Debskij thanks for the comment. I changed the inference according to the documentation:
https://docs.openvino.ai/latest/notebooks/002-openvino-api-with-output.html